website/sidebars.js

```diff
@@ -910,6 +910,29 @@ const sidebarSettings = {
       ],
     },
   ],
+  sqlFunctions: [
```

Comment:
Is the plan to have an index page that shows all glossaries (that we link to in the `learn` dropdown), or link to both the Analytics Engineering glossary + SQL glossary?

And I'm assuming, bc these SQL glossary pages exist outside of the AE glossary framework, they're not eligible to be used as `terms` throughout the docs site?

Reply:
Originally, I intended them to be separate, but I think having them both in the `/glossary` is a good idea. I need to play around with this and find a way to make it so the AE glossary can stay in the `/terms/` folder and the SQL glossary stays in the `/sql-glossary/` folder. Ideally, I wouldn't want the SQL glossary entries to be in the `/terms/` folder.